x/distribution/types/params.go

```diff
@@ -18,6 +19,10 @@ func DefaultParams() Params {
 
 // ValidateBasic performs basic validation on distribution parameters.
 func (p Params) ValidateBasic() error {
+	if p.CommunityTax.IsNil() {
```

Comment:
Why not using just doing `return validateCommunityTax(p.CommunityTax)` instead?

Reply:
That's a good idea. I'll add an update